test/integration/features/addon-xpaas.feature

```diff
@@ -19,18 +19,18 @@ which are then available in OpenShift to the user.
 
   Scenario Outline: User deploys, checks out and deletes several templates from XpaaS imagestream
    Given Minishift has state "Running"
-    When executing "oc new-app <template-name>" succeeds
-     And executing "oc set probe dc/<service-name> --readiness --get-url=http://:8080<http-endpoint>" succeeds
```

Comment:
We don't need to set a readiness probe anymore?

Reply:
In a confusion i removed it. In the next commit i will add it.

Reply:
@amitkrout You can update this PR and add back the readiness probe.

Reply:
Done